config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1beta1.yaml

```diff
@@ -333,6 +333,9 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
+        env:
+          - name: TEST_WINDOWS
```

Comment:
this requires k8s 1.22+, are we testing with 1.22 in the release-1.1 branch?

Reply:
Looks like it 
From the `pull-cluster-api-provider-azure-ci-entrypoint-v1beta ` job on https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2027


NAME                              STATUS   ROLES                  AGE     VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION     CONTAINER-RUNTIME
capz-uwrg2l-control-plane-qtbvc   Ready    control-plane,master   2m10s   v1.22.1   10.0.0.4      <none>        Ubuntu 20.04.3 LTS   5.8.0-1039-azure   containerd://1.5.4
capz-uwrg2l-md-0-thklx            Ready    <none>                 52s     v1.22.1   10.1.0.4      <none>        Ubuntu 20.04.3 LTS   5.8.0-1039-azure   containerd://1.5.4
capz-uwrg2l-md-0-wmdnh            Ready    <none>                 31s     v1.22.1   10.1.0.5      <none>        Ubuntu 20.04.3 LTS   5.8.0-1039-azure   containerd://1.5.4


https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/2027/pull-cluster-api-provider-azure-ci-entrypoint-v1beta1/1488575602268049408

Reply:
good point. doesn't look like it https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/61fcf75d5a290577be64604db363baf6b5e604bb/hack/create-dev-cluster.sh#L50